recipes/oscars/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "oscars" %}
+{% set version = "2.1.8" %}
+{% set sha256 = "c37853089db601537c48484ee72b84ae40893daf0d4b86030e4bf080dc3965b7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dhidas/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: True  # [py<36]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - matplotlib-base
+    - mpi4py
+    - numpy
+    - scipy
+
+test:
+  imports:
+    - oscars.sr
+    - oscars.th
+    - oscars.parametric_surfaces
+    - oscars.plots_mpl
+    - oscars.plots3d_mpl
+    - oscars.util
+
+about:
+  home: http://oscars.bnl.gov/
+  license: BSD-3-Clause
+  license_family: BSD
```

This actually looks more like the [EPICS](https://epics.anl.gov/license/open.php) license (though with a minor extra clause).
```suggestion
  license: EPICS
  license_family: BSD
```

---

Sounds reasonable, thanks!